libraries/chain/platform_timer_bsd.cpp

```diff
@@ -0,0 +1,116 @@
+#include <eosio/chain/platform_timer.hpp>
+#include <eosio/chain/platform_timer_accuracy.hpp>
+
+#include <fc/time.hpp>
+#include <fc/fwd_impl.hpp>
+#include <fc/exception/exception.hpp>
+#include <fc/log/logger_config.hpp> //set_os_thread_name()
+
+#include <mutex>
+#include <thread>
+
+#include <sys/types.h>
+#include <sys/event.h>
+#include <sys/time.h>
+
+#ifndef KEVENT_FLAG_IMMEDIATE
+#define KEVENT_FLAG_IMMEDIATE 0x001
```

Comment:
Isn't used. Looks like a holdover from trying to make macos & bsd impls shared? I think it's worth trying to find a way to make them shared since they do seem really similar -- something a couple `ifdefs` might be able to resolve?

Reply:
Yes, absolutely agreed, I am already working on single kevent based file for both osx/bsd, since they are very similar. It was just easier to play with separate file while debugging. Thank you!